### PR TITLE
Removes botany's superior chem dispenser

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request

- Removes botany's chem dispenser from the game
- reverts https://github.com/BeeStation/BeeStation-Hornet/pull/1560
## Why It's Good For The Game

Departments need to depend on each other. Progression is meant to take time, and is reduced when departments work together.

This item was created since rounds were short, and botany _needed to get their endgame faster_. 

>The problem: Beestation rounds are SHORT. Like, 25 minutes short at times. An effective Botanist has to speed run tech storage or chemistry to steal a dispenser (optimally) in order to get chemicals needed to mutate plants and increase yield.

However, this has caused Botany to no longer depend on the rest of the station to reach their endgame. No longer does Botany need to ask chemists for materials, or suffer when no chemists or doctors are around to press buttons.

Rounds should be longer, not optimized for a rush to the end.
---

Why ask a chemist for anything, when your private dispenser is outright **better** than theirs?

image

image

## Changelog
:cl:
del: Removed botany's specific chem dispenser.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
The real reason we added this is because at the time TG had just nerfed botany so I thought it'd be funny to buff it.

**Comment by bloons3** [comment]
> The real reason we added this is because at the time TG had just nerfed botany so I thought it'd be funny to buff it.

🧛

**Comment by ToasterBan** [comment]
I think due to chem factories a chemist will most likely just give a botanist a plumbing constructor and some ducts

**Comment by Effemu** [comment]
^do you honestly expect chemists to cooperate with anyone

**Comment by bloons3** [comment]
> ^do you honestly expect chemists to cooperate with anyone

That's how it used to be in the before times.

Legends tell of wars waged between botanists and chemists, of windows battered down and chemical dispensers stolen, beakers of mutagen handed over at hatchetpoint, and the fabled raiding of tech storage.

**Comment by ghost** [comment]
People forgot that not having such a dispenser is not an obstacle for botany. They can grow their own chems with minimal effort without any cooperation. It will add a few minutes to such hermits, at most.

**Comment by ToasterBan** [comment]
do what you did what bs miners and make it a config wether botany gets it or not

**Comment by Chessus** [comment]
Oh god no. Those pre-dispenser days were awful. Zero fun. I agree with you in principle, Interdepartmental is a neat **idea** but in **practice** on a public, non-whitelisted lrp server(~sage) it does not work.

**Comment by Archanial** [comment]
What about sprites?

**Comment by Cenrus** [comment]
Our version of hydro is FAR too reliant on chemicals and I really wish it wasn't. Making potency 100 plants should require more effort than waterboarding them with 200 units of Saltpetre.

**Review by ghost — DISMISSED** [review]

**Comment by Sinestia** [comment]
> 
> 
> do what you did what bs miners and make it a config wether botany gets it or not

2022

game_options.txt is a 50000 line file because people only want to skim the surface of tumors instead of just excising them entirely

**Comment by OceanFish1** [comment]
Botany mains are crying and shaking rn.

This will return botanists a reason to roundstart tide into chemistry to steal one dispenser.

**Comment by francinum** [comment]
> What about sprites?

What about them? The Lifegivers still have their dispenser.

**Comment by TeomanTheGreat** [comment]
The Botany mains will just steal chemistry's dispensers again. I like where this pr is trying to go but it just won't work.

**Comment by bloons3** [comment]
> The Botany mains will just steal chemistry's dispensers again

Good

**Comment by francinum** [comment]
> > do what you did what bs miners and make it a config wether botany gets it or not
> 
> 2022
> 
> game_options.txt is a 50000 line file because people only want to skim the surface of tumors instead of just excising them entirely

I agree with this post in theory, I'd love to just lance some boils and call it a day.

However, I acknowledge that the botany mains have grown accustomed to sucking on the teet of terrible, coddling game design that they will do anything to get that drip-feed of low effort plant waterboarding substance back.

**Comment by bloons3** [comment]
> > > do what you did what bs miners and make it a config wether botany gets it or not
> > 
> > 
> > 2022
> > game_options.txt is a 50000 line file because people only want to skim the surface of tumors instead of just excising them entirely
> 
> I agree with this post in theory, I'd love to just lance some boils and call it a day.
> 
> However, I acknowledge that the botany mains have grown accustomed to sucking on the teet of terrible, coddling game design that they will do anything to get that drip-feed of low effort plant waterboarding substance back.

If Botany wants to start a war with Medical, it'll be just like old times.

**Comment by heepox** [comment]
back to stealing from tech storage it is? or just asking engi/ai/cap for it.

basically this was added for shits and giggles, but also stopped botany tide. removing will just bring back botany tide.

**Comment by LivingBatteryBox** [comment]
Honestly botany tide isn't fun to do or deal with. If you want to nerf botany just make it so that you can dispense reagents to make saltpetre and the like without actually being able to directly dispense them. Basically make it a chemical dispenser that is more limited instead of being more powerful.

**Comment by Sinmero** [comment]
>Legends tell of wars waged between botanists and chemists, of windows battered down and chemical dispensers stolen, beakers of mutagen handed over at hatchetpoint, and the fabled raiding of tech storage.

It was horrible and it was one of the reasons botany have their own dispencer now.
Dont expect chemists to make anything for you at all.
Botanists can make their own reagents from plants argument doesnt work because it didnt prevent the botany tide before they had their own dispencer.
Botany tide is absolute cancer.

**Comment by Cartlord** [comment]
Please, for the love of god, DON'T do this. I play on Golden, so I don't know what kind of spiffy highpop you have on Sage where you can actually have departments co-operating, but this will leave Golden botanists with basically no choice but to break into Chems and steal a dispenser because there'll either be 0 chemists online or one chemist who doesn't care. This seems to be something that'd be wonderful for highpop, but Golden doesn't HAVE highpop.
Quite frankly, I think the botany chemical dispenser could use a buff, so it can produce Ammonia - and therefore heal plants, as well as buffing Yield (which, in my experience, saltpetre DESTROYS) from roundstart.

**Comment by PowerfulBacon** [comment]
Botany still depend on science for upgrades

**Comment by eeSPee** [comment]
Compensate for this by adding mutagen purchases in cargo?

And give them some starting unstable mutagen to start. Maybe bottles in the vending machine?

I really don't like it when a department is locked behind other departments and cannot do its basic functions.

**Comment by francinum** [comment]
> Compensate for this by adding mutagen purchases in cargo?
> 
> And give them some starting unstable mutagen to start. Maybe bottles in the vending machine?
> 
> I really don't like it when a department is locked behind other departments and cannot do its basic functions.

I'm perfectly fine with this, honestly.

**Comment by utilbird** [comment]
> I'm perfectly fine with this, honestly.

Meh, it can ruin the enjoyability of a department, especially on lrp. Encouraging people to get help from other departments is great but forcing it makes it a chore for both parties.

**Comment by utilbird** [comment]
Botany dispenser could probably use a default charge reduction. Removing it will just lead to more tech storage doors being hacked. It's pretty poor design if a job requires ~5-10 minutes of preparation so they can (enjoyably) play.

**Comment by Sinestia** [comment]
one patch you could add is giving some UM in the vendor

like 10-20 bottles. As well as cargo

it would also be nice if plants actually processed chems in a manner like they do on /tg/, that way it can be tweaked in the ideal of making it so that yes, you can do your job comfortably, but you still REALLY want things from other people. But alas we lack that and all that's really there is mutation odds

**Comment by ToasterBan** [comment]
3 bottles for 800 bucks? i normally use up 3 bottles just to get a mutation

**Comment by bloons3** [comment]
Crates cost $500, so it's really 300.

**Comment by ToasterBan** [comment]
maybe instead of deleting the dispenser you remove it from roundstart maps and have be printable from service lathe after botanical tech has been researched? this makes it need a normally never researched tech

**Comment by PowerfulBacon** [comment]
🔑

**Comment by jlsnow301** [comment]
Delete it who could have bountied such a thing

**Comment by Isy232** [comment]
Here is my idea:

- Make botany less reliant on chemicals to get the cool stuff, make it just another way. 

- Alternatively, Give them a mutagen plant they could farm and use their grinder with to make their own mutagen.

Otherwise botanists are forced to do one of three:
1: Go to chemistry and have a chemist who will actually help them. I think on MRP this will actually be a decent majority of rounds, but there will still be some where the chemists are helping deal with medicine or working on their own project. Or maybe there just are no chemists.
2: Mix it yourself in chemistry. Get the AI to let you in or get access from HoP, however this is very annoying as they now have to waste a lot of time walking back and forth.
3: Make your own, get into tech storage one way or another and get the parts yourself. This requires getting tools and take a fair bit of time, especially if you have to do it every fucking round.
4: Steal a chem dispenser, for the lazy fucks who wont build one themselves. This usually ends with chemistry using _fun_ grenades on hydroponics.

**Comment by pickleinjar** [comment]
this is kinda cringe ngl

**Comment by BeloneX** [comment]
> this is kinda cringe ngl

Don't make comments if you don't have anything of value to say.

**Comment by hotimep** [comment]
i still remember botanists stealing the chem dispenser because they ran out of unstable mutagen

**Comment by bloons3** [comment]
The more I look at Botany's machine, the worse it is.
Let me list the ways:
1. Saltpetre (1 part Potassium, 1 part Nitrogen, 3 parts Oxygen) - Normally results in 3 units, Botany gets 5 per click. Requires no effort to mix.
2. Unstable Mutagen - 1 part Chlorine, 1 part Phosphorus, 1 part Radium. Not really bad, but Botany should get the raw reagents not the finished product.
3. Pest Killer - 1 Toxin, 4 Ethanol. Chemistry cannot mass product this, since they have to get Toxin from a Medical Vendor. This is a STRAIGHT upgrade over chemistry.
4. Plant-B-Gone - 1 Toxin, 4 Water - Same issue as Pest Killer.
5. Weed Killer - 1 part Toxin, 4 parts Ammonia - Same issue as Pest Killer.
6. Diethylamine - 1 part Ammonia, 1 part Ethanol - No effort to mix (Normally chem dispenser macro)

3 of these straight up make Chemistry/Medical/Cargo useless since Botany doesn't require materials outside of their department. 

If this machine took Cargo Supply Packs to refill like the other vendors did, it wouldn't be that bad.

**Comment by Sinmero** [comment]
>3 of these straight up make Chemistry/Medical/Cargo useless since Botany doesn't require materials outside of their department.

Botany can still order bees from cargo. Relying on chem to be able to do your job properly is bad. Chem usually wont do anything for you.

**Comment by bloons3** [comment]
> Relying on chem to be able to do your job properly is bad.

It's not relying on chem, it's being reliant at ALL on materials which don't automagically renew. Botany already has a machine which shits out endless supplies of cheap chemicals, it's called the "NutriMax". ( I actually can't believe that the magical botany machine deprecates ROUNDSTART Botany's own vendors)

image

If you use up those materials, you have to repurchase them at Cargo.

Cargo is a great place to exchange money for things you don't care to make yourself. 

---

To put it differently, you shouldn't be able to easily reach your own department's endgame without interacting with another department.
It is possible with time and effort for Botany to make very powerful plants without using mutagen, manually tilling the soil, smashing bugs, and all the rest. 
Having access to other departments that can easily (for the department) do helpful things like make chemicals or provide advanced/upgraded machines speeds Botany up.

**Comment by PowerfulBacon** [comment]
Vote at 15/57, safe to say its not going to pass
